### ISSUE TITLE
Rebalance Iron Balls Satellites properly

### ISSUE THREAD

**Issue by TaipionsSalemClient** [PavelZinchenko/event-horizon-frontier-builds#61]
With the current 3x3 L slot on it, it is a joke, similar to a HoverTank, except the HT can at least fit an AMR while the iron ball can't.

What is the purpose of a capital-sized satellite, if it can't fit any good L type weapon?

There is only one fair solution to this:
Design capital satellites for each satellite type, not necessarily all at once, but with that goal in mind.

The "old" HT should become "omni slot" for compatibility and be the "barrel"-equivalent of capital satellites,
which means the other satellites for their respective slot types should have no engine slots (already correct for iron ball),
other weapon-satellites should be made with the intention to enable them to use (almost) all the good weapons of that respective category.

Example: Dart Mk3 has a 2x4 C slot that enables usage of most good C weapons (except 3x3 Holy Cannon), so a capital "Dart" Satellite should not be worse when it comes to weapon choices.

(...and not to forget a capital variant of the "pot" satellite)

**Comment by TaipionsSalemClient** [comment]
@Inedilarion
@Zen9890
Please elaborate why you disagree, throwing "thumbs down" and not writing anything is a cheap move.

**Comment by Inedilarion** [comment]
https://github.com/PavelZinchenko/event-horizon-frontier-builds/issues/41

**Comment by Zen9890** [comment]
> @Inedilarion @Zen9890 Please elaborate why you disagree, throwing "thumbs down" and not writing anything is a cheap move.

This game has infamously long period of bullshit meta that is not over yet, mounting best weapon in the game on a satellite and having nicely arranged 8 2x2 blue/green slots just outclassed any other satellite option for Capital ships and battleships (as of recent update with new satellite-related perk)

**Comment by TaipionsSalemClient** [comment]
> #41

ok, so your opinion is just based on false assumptions, good,
you just need to see this clearly, then we can discuss this in an objective way

Number one: HLC is not the "best weapon in the game", and on top of that only deals 1 dmg type, if you see everyone using HLCs you can simply add more energy resists to your ship and/or use the energy resist perk, and/or use worm/jorm ships to nullify the damage.
It is one of the best weapons for medium range if energy damage is acceptable, that's it.

This is gonna be the same story as the DR, which is now defunct and utterly useless == a weapon/item in the game that is not used, because it has been wrongly nerfed to oblivion.

Number two: The iron ball only appeared too strong in its class due to the lack of other capital satellites except for the HoverTank, if you had read my proposal, you would have gotten that point.

**Comment by TaipionsSalemClient** [comment]
> > @Inedilarion @Zen9890 Please elaborate why you disagree, throwing "thumbs down" and not writing anything is a cheap move.
> 
> This game has infamously long period of bullshit meta that is not over yet, mounting best weapon in the game on a satellite and having nicely arranged 8 2x2 blue/green slots just outclassed any other satellite option for Capital ships and battleships (as of recent update with new satellite-related perk)

Yes, outclassed any other satellite, like the HoverTank outclassed every other satellite before as it simply was the only capital satellite and rightfully out-classed every other satellite as they were all of a lower class.
If this does not make sense to you, then you may want to re-read the whole thing, because that's how it's supposed to be, full stop.

Now the iron ball appeared to "outclass" every other, because it happened to be, in most cases, better than the ONLY OTHER satellite in its class, the HoverTank.

If you had read my proposal in full, you would have noticed that I suggested balancing the capital satellites with a (soon to be) full range of capital satellites in mind, as it would be very stupid to now nerf the L-type-capital satellite (that's the iron ball!) because it appears "too strong", only to buff it later when the other Satellites are added, and it suddenly appears too weak.

Again: There should be a full range of satellites, which is definitely not too much to ask for, having a capital version of each type, whereas the HoverTank would be the "barrel mk4" as it has engine slots, which the others should not have.
I mean, ok, that's up to the dev, capital satellites might deviate from the sub-capital satellites in one way or another, but they should not be weaker, and a 3x3 L slot is just that, very weak.

**Comment by Inedilarion** [comment]
> > #41
> 
> ok, so your opinion is just based on false assumptions, good, you just need to see this clearly, then we can discuss this in an objective way
> 
> Number one: HLC is not the "best weapon in the game", and on top of that only deals 1 dmg type, if you see everyone using HLCs you can simply add more energy resists to your ship and/or use the energy resist perk, and/or use worm/jorm ships to nullify the damage. It is one of the best weapons for medium range if energy damage is acceptable, that's it.
> 
> This is gonna be the same story as the DR, which is now defunct and utterly useless == a weapon/item in the game that is not used, because it has been wrongly nerfed to oblivion.
> 
> Number two: The iron ball only appeared too strong in its class due to the lack of other capital satellites except for the HoverTank, if you had read my proposal, you would have gotten that point.

Quoting you: "ok, so your opinion is just based on false assumptions, good, you just need to see this clearly, then we can discuss this in an objective way"

Not a single satellite in the game, except the Hovertank, can replace a ship's gun slot due to the inability to install a powerful cannon with a autotarget. Darth M3 allows you to install only an ion cannon without autotarget, which is much weaker than a heavy lightning cannon, and railguns, which have even less DPS and projectile speed. But the hovertank not only has engines, armor and green cells, but also provides two turrets with holy cannons, which are more powerful and more convenient than the basic weapons of half the titans (and the only thing that prevents these satellites from completely replacing the weapons of the titans in the frontier is the low projectile speed HC, making it ineffective. In EH RPGs, by the way, there is no such problem, and a hovertank with HC is an attribute of almost all titan builds, which is quite sad).

Satellites should make the ship stronger, but should not replace the ship, making all ships the same (no matter how much you would like to get imba so as not to think about different builds). Isn’t that why there is such a variety of ships and weapons in the game? The satellites should be auxiliary, which is what all the M1-M3 satellites are, but nothing more. If the Iron Ball didn't originally have a heavy lightning cannon turret, it would be an excellent support companion like Pot. But it had this turret, which, even without all the surrounding blue-green cells, would have broken the individuality of all the titans and would have become a meta for the EHF

Regarding the full line of satellites, if we take the existing M1, M2 and M3 as an example, then satellites with weapons should not have a turret and there should not be a convenient place for armor and modules. And satellites for engines and modules should not have weapons. In general, neither the Hovertank nor the Iron Ball fit into such a “complete series”

**Comment by TaipionsSalemClient** [comment]
ok, let's quote this properly:


#1 you @Inedilarion said: "_Not a single satellite in the game, except the Hovertank, can replace a ship's gun slot due to the inability to install a powerful cannon with a autotarget_"
=> How is that even an argument? I just explained it, the HoverTank **IS A CAPITAL SATELLITE!** ...and so is the iron ball,
I have no idea why you refuse to accept this, **THEY ARE CAPITAL SATELLITES!** ... -.-
... this means **THEY ARE SUPPOSED TO BE BETTER THAN NON-CAPITAL SATELLITES!**
(seriously, if you failed to understand this basic point, please refrain from answering this any further, you won't do yourself a favour if you do, nor anyone else...)

#2 you @Inedilarion said: "_Darth M3 allows you to install only an ion cannon without autotarget, which is much weaker than a heavy lightning cannon_"
Are you aware that you are...
... comparing an L-type satellite with a C-type satellite?
... comparing a battleship-satellite WITH A CAPITAL SATELLITE?!?!
... you are intentionally forgetting to mention the HIC, which is the strongest of the possible (mid to long range) weapons equipable in a dart mk3?
=> you do not want to lay out the facts straight
=> you only want to look at what supports your (wrong) view and ignore all else

#3 you @Inedilarion said:  "_Isn’t that why there is such a variety of ships and weapons in the game_?"
Where have you been to protest against the omni perk?!? How dare you say there is variety in guns/ships with the omni perk literally allowing every ship to equip every  weapon type, but complain that a satellite allows ships to equip L type weapons?!?
...seriously?!

TL;DR:
- your "facts" are all wrong
- like really, really wrong
- you try to impose your personal opinion on others here, ignoring the facts and valiantly misinterpreting everything, ignoring anything that does not suit your view

I have only one question, as this all (your view on it) does not make any sense at all:
Why?
Why do you hate this so much?!
You know we should have a full range of capital satellites at some point in the future, and the "old" iron ball is exactly how all of those should be, what's your super problem with that?

**Comment by TaipionsSalemClient** [comment]
I'll cook it down a little more here, your concerns are:
- you personally dislike the "old" iron ball
- you believe it's a problem it can equip HLC because HLC is too strong => so you successfully nerfed the iron ball because you think HLC is too strong => this is as wrong as it gets

TL;DR:
HLC too strong => nerf iron ball
**THIS IS WRONG**, if you think HLC is too strong, then deal with it, not with the satellite!

**Comment by Tree3324417** [comment]
To add on this, after some discussions on the Event Horizon Official Discord, we have came to a possible change to the Iron Ball that keep it's strength (Field HLC, HLC balance is another field that i will not talk about here), while not dwarfing Hovertank in support potential:
-GIves it back the old shape and layout, but replace all of the green-blue space with blue (armor) space, allowing it to field HLC or tachyon (HLC balance is another thing that is not that relevant to here), but not dwarfing Hovertank's support strength (which means now iron ball will support HP a lot, while hovertank gives green and yellow space

**Comment by Zen9890** [comment]
> as a consensus from the discord discussion, the following suggestion:
> 
> 1.) bring back "old" iron ball shape 2.) change half the green slots to blue (every second 2x2 to blue), for a total of 4 2x2 blue and 4 2x2 green squares 3.) keep the HLC discussion out of this, it's a different issue

This brings into the table semi-freshly buffed tachyon cannon, and converting all capital/battleships into some kind of DRacle parody is questionable, overall yes, there isnt anything inherently bad with your suggestion, but if we want 1 satellite not to dominate entire meta we should get (again, as you said) more satellites of each type, and then get proper balance of weapons so anything is useable in its own way, not just "haha this weapon has too good stats, i will oblitirate everything with 0 braincells now", so i'd also personally remove targetting from both hovertank and crystal altogether since this also promotes braindead turret builds, or just make it not as extreme (maybe like, frontal targetting at best?)

**Comment by TaipionsSalemClient** [comment]
players WILL use the items they perceive as "best", with the discord and arena people see what others use and what works,
and thereby a "meta" is generated, if it's not HLCs or tachys, it is something else,
this is the nature of such games and not something you can design or balance a single item (the iron ball satellite) around

**Comment by SuplexCity25284** [comment]
I haven't even found iron ball satellite, when does it become available?  I'm currently in the 140s as far as lvl.
